### DISCUSSION TITLE
api status in UI

### DISCUSSION THREAD

**Discussion by gamebeaker** [firefly-iii/firefly-iii#9001] [General]
Hi,
in the docs is it mentioned, that firefly is not using the API exclusively. I wanted to ask what the progress is in this regard. (No percentage more like a lot of work to do or if it is missing minor changes.)
![grafik](https://github.com/firefly-iii/firefly-iii/assets/9269847/b3cab977-3f46-4528-b9e9-c8f9cb7542fb)
And if it is planed to seperate the database + api and the UI in github.

**Comment by JC5** [comment]
Lots of work I'm afraid but yea this is planned. No ETA, sorry
